### PR TITLE
Create proper contributors section

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

**Comment by gmandyam** [comment]
See https://www.rfc-editor.org/rfc/rfc7322#section-4.11, taking note of the "may" requirements:

"The Contributors section may include brief statements about the nature of particular contributions ("Sam contributed [Section 3](https://www.rfc-editor.org/rfc/rfc7322#section-3)"), and it may also include affiliations of listed contributors.  At the discretion of the author(s), contact addresses may also be included in the Contributors section, for those contributors whose knowledge makes them useful future contacts for information about the RFC."

In other words, the lack of contact information for contributors is perfectly acceptable as per RFC 7322.  The existing text is already "proper".

**Comment by cabo** [comment]
Yes.  
But this PR was not about RFC 7322 minimum requirements, but about a more prominent and useful presentation of the list of contributors, in a way that is also closer to the list of authors.
(I didn't include "brief statements about the nature of particular contributions" because that would then be added by the authors, using the `contribution` YAML field, if desired.)

**Comment by gmandyam** [comment]
We are tracking PR's against issues (as much as possible).  If you have an issue with the style adopted in the Contributors section then please raise an issue first so that it can be discussed.

**Comment by cabo** [comment]
The issue is #236

**Comment by henkbirkholz** [comment]
Hi. I am summoned here because my name is used on github.

I just stared at the head of master https://github.com/ietf-rats-wg/eat/blob/master/draft-ietf-rats-eat.md#contributors. This PR is an improvement.

"Contributors" is a structure that needs specific kramdown syntax to really take effect. Current master includes simple text content. This PR improves that simple text to kramdown that enables xml2rfc to create a dedicated contributor section as intended by RFCed.

Hence, I'd like to +1 this suggested change as it proposes a better way of including my name :-)

**Comment by henkbirkholz** [comment]
If this PR needs it's own dedicated issue, I can open a new issue. But is that really necessary?

**Comment by gmandyam** [comment]
Go back to RFC 7322:

" The format of any contact information should be similar to the format of any information in the Author's Address section."

Proposed PR does not satisfy the above, as it is not consistent with "format".  Therefore it is not really "proper".  This is another reason why I left out the contact info in the Contributors' section, as RFC 7119 allows for exclusion.

**Comment by gmandyam** [comment]
> If this PR needs it's own dedicated issue, I can open a new issue. But is that really necessary?

Yes it is.  Draft is in WGLC.  All unsolicited PR's should be tracked with corresponding issues so that they can be matched against WGLC comments.

**Comment by henkbirkholz** [comment]
Done. See https://github.com/ietf-rats-wg/eat/issues/276

Please mind, this is an editorial suggested change.

**Comment by cabo** [comment]
> Go back to RFC 7322:
> 
> " The format of any contact information should be similar to the format of any information in the Author's Address section."

Indeed, this is part of the point of this PR.

> Proposed PR does not satisfy the above, as it is not consistent with "format". Therefore it is not really "proper". This is another reason why I left out the contact info in the Contributors' section, as RFC 7119 allows for exclusion.

I made the PR specifically because there has been a clear trend in the eight years since RFC 7322 was written towards Contributors sections that look like authors' addresses sections.  RFC 7119 is about IPFIX and probably not relevant here.

Please stop arguing around the slightly sloppy name of this PR (you think "proper" means compliant to old documents, I think "proper" means up to current best practices).

I'm really not used to this level of resistance to informed advice.

**Comment by gmandyam** [comment]
>Please stop arguing around the slightly sloppy name of this PR (you think "proper" means compliant to old documents, I think "proper" means up to current best practices).
>I'm really not used to this level of resistance to informed advice.

Didn't know that open discussion amounted to "arguing".  I will quite commenting if the discussion is perceived as hostile.  My apologies.

**Comment by cabo** [comment]
Meant argue [1], sorry.

ar·gue | ˈɑrɡju |
verb (argues, arguing, argued)
1 [reporting verb] give reasons or cite evidence in support of an idea, action, or theory, typically with the aim of persuading others to share one's view: [with direct speech] : “It stands to reason,” she argued | [with clause] : defense attorneys argue that the police lacked “probable cause” to arrest the driver.
• [with object] (argue someone into/out of) persuade someone to do or not to do (something) by giving reasons: I tried to argue him out of it.
2 [no object] exchange or express diverging or opposite views, typically in a heated or angry way: figurative : I wasn't going to argue with a gun | [with object] : she was too tired to argue the point | don't argue with me.

**Comment by gmandyam** [comment]
Sure.  Let's merge.  Any issues with style will be covered in IESG review and we can revert if necessary.

**Comment by cabo** [comment]
Thank  you.

**Comment by gmandyam** [comment]
This PR is already merged, but my responses earlier were rather curt. I apologize for that - I was multitasking at the time. I feel I should provide a more well-thought out response to this well-intentioned PR.

a)  RFC 7322 is the current RFC style guide for editors.  If there is any authoritative guidance stating that parts of it can be ignored or interpreted as advisory at best (beyond the extension cited in the header material of RFC 7322), then I certainly would welcome a pointer.

b) Even though the current RFC was published in 2014, it is still being applied to candidate RFC's during broader IESG review based on my recent experience.  For instance, RFC 9223 (https://datatracker.ietf.org/doc/rfc9223/) is an informational draft that was held up from publication for 4 years from my initial submission primarily due to issues related to style adherence.  Ironically at the time of my original submission, there were already many products on the market implementing this specification (thus meeting at least the "running code" criterion described in https://www.rfc-editor.org/rfc/rfc7282#page-3).  

c)  In short, candidate RFC's are held to the standard of the Style Guide, and I don't believe the EAT editors have significant discretion in that regard.  If certain individuals have their personally-preferred style then I recommend they validate any proposed PR against RFC 7322 before pushing it to the main repo.

d)  With regards to a Contributors section, the original text was conformant with the RFC Style Guide based on my assessment.  This  particular PR is no conformantt, in my interpretation.  Specifically as per (https://www.rfc-editor.org/rfc/rfc7322#section-4.11 (Contributors Section):  "The format of any contact information should be similar to the format of any information in the Author's Address section".

Going to the Author's Address section:  https://www.rfc-editor.org/rfc/rfc7322#section-4.12:  "Contact information must include a long-lived email address ...".

Since I could not satisfy the 'must' requirement above as I did not have long-lived email addresses for all contributors, I chose not to include contact information in the Contributors section as originally merged into the draft.  This was allowed by RFC 7322 as well.

All that being said, I am not claiming that what is in the current Style Guide is something I particularly care for, and I would not be unhappy if in the future that candidate RFC editors would be free to ignore it either partially or fully.  But if adherence gets us closer to completion of the EAT specification (also now 4 years old and still not an RFC) then I think it is certainly worth the sacrifice.

**Comment by cabo** [comment]
Hi Giri,

RFC 7322 is still the valid style guide to apply, together with other information you find listed at https://www.rfc-editor.org/styleguide/.

RFC 7322 gives significant lenience on the contributor section, as that was nascent at the time.  The MUST on author address information combined with the SHOULD on contributor contact information leads to a SHOULD.

My assumption was that the other contributors may want to supply affiliation and contact information as well.  But that can be done now, after merging of the initial PR, so they can see which authoring format can be used for that.

I was also expecting the authors (editors) to potentially supply a line or two what the contribution was, but that is entirely optional.

These are all implicit to the way we do modern contributor sections, so I'm sorry for not making these points explicit.

**Comment by cabo** [comment]
Since I'm working on this essentially (except for an RTL issue) completed RFC-to-be right now, here is just one example for what I mean with "modern contributor section":

https://www.rfc-editor.org/rfc/authors/rfc9290.html#name-contributors

In this case, both listed contributors provided an email address; one also provided a URI that makes it easier to find his pertinent work.  The contributions are clearly described in short sentences (which was easy here).

**Comment by gmandyam** [comment]
>RFC 7322 gives significant lenience on the contributor section, as that was nascent at the time. The MUST on author address information combined with the SHOULD on contributor contact information leads to a SHOULD.

That wasn't the way I read it.  

The actual language is "should be similar to the format of information in the Author's Address section", which leaves some room for interpretation.  Therefore (in my interpretation) the must requirement for the author's (or in this case contributor's) contact email is a requirement, if the contact info format that is being adopted is what is described in Sec. 4.12.  Since the PR leverages the kramdown-rfc2629 attributes corresponding to author address which are consistent with style guide, this means that all must requirements for contact information should be followed.

Independent of that, I don't think we want to finalize the contributors' section without obtaining the minimum contact information for all contributors (along with contributions).  This practice appears to have been followed in https://www.rfc-editor.org/rfc/authors/rfc9290.html#name-contributors, so I assume you agree.
